lib/stat_tracker.rb

```diff
@@ -447,5 +447,40 @@ def team_info(team_id)
     team_info
   end
 
+  def away_games_only 
+    away_games = @game_teams.find_all do |game_team|
+      game_team != game_team.hoa
+    end
+    away_games
+  end
+
+  def away_create_team_goals_and_games
+      team_goals_and_games = {}
+      away_games_only.each do |away_game| 
+        team_id = away_game.team_id
+
+        team_goals_and_games[team_id] ||= { goals: 0, games: 0 }
+
+      team_goals_and_games[team_id][:goals] += away_game.goals
+      team_goals_and_games[team_id][:games] += 1
+    end
+      team_goals_and_games
+    end
```

Comment:
Maybe if we have time we can put this logic into a module since we use it a lot throughout the project!